scripts/dev.sh

```diff
@@ -0,0 +1,539 @@
+#!/bin/bash
```

Comment:
This doesn't seem to be checked in with execute permission. Might be helpful to do so.